powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -179,4 +181,12 @@ private double samplingRate(final PowertoolsLogging powertoolsLogging) {
     private Logger logger(final ProceedingJoinPoint pjp) {
         return LogManager.getLogger(pjp.getSignature().getDeclaringType());
     }
+
+    private static Optional<String> getXrayTraceId() {
+        final String X_AMZN_TRACE_ID = SystemWrapper.getenv("_X_AMZN_TRACE_ID");
+        if(X_AMZN_TRACE_ID != null) {
+            return Optional.ofNullable(X_AMZN_TRACE_ID.split(";")[0].replace("Root=", ""));
```

Comment:
Just nit pick but import static ?

Reply:
sometimes i feel like it makes it explicit. But, I like your nit pick, i'll make it static import :)